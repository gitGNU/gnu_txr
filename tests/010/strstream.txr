@(do 
   (defun lazy-byte-stream (s)
     (let (ch) (gen (set ch (get-byte s)) ch)))

   (let* ((data "春が来た (Haru-ga Kita/Spring has Come)")
          (in-byte (make-string-byte-input-stream data))
          (out-byte (make-string-output-stream)))
     (each ((b (lazy-byte-stream in-byte)))
       (put-byte b out-byte))
     (print (get-string-from-stream out-byte))
     (pprint "\n")))
