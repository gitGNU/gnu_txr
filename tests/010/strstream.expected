"春が来た (Haru-ga Kita/Spring has Come)"
